Src/BootCamp.Chapter/Lesson3.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace BootCamp.Chapter
+{
+    class Lesson3
+    {
+        static string name = GetName();
+        static int age = GetAge();
+        static float weight = GetWeight();
+        static float height = GetHeight();
+        static float bmi = GetBMI(weight, height);
```

Comment:
CalculateBmi is a better name. Getting refers to retrieving something that is complete, not computing.